socorro/processor/symbol_cache_manager.py

```diff
@@ -5,27 +5,37 @@
 import os
 import sys
 import tempfile
-import threading
-from datetime import datetime
-from collections import defaultdict
 from ordereddict import OrderedDict
 
 from configman import Namespace, RequiredConfig
-from configman.converters import class_converter, timedelta_converter
 
-from socorro.lib.datetimeutil import utc_now, UTC
-import pyinotify
 
+if os.uname()[0] != 'Linux':
+    # You're not on Linux! Avoid pyinotify like the plague
+    class ProcessEvent(object):
+        # Defining a class means we can't define the EventHandler class
+        # without indenting the whole thing in an if-block.
+        pass
 
-#==============================================================================
-class EventHandler(pyinotify.ProcessEvent):
-    #--------------------------------------------------------------------------
+    import warnings
+    warnings.warn(
+        'ProcessEvent is disabled on operating systems that does not '
```

Comment:
Who deploys on non-Linux anyway :)

I actually don't know what to make of this comment. My intention was just around development, not about use or deployment. Now you're making me un-confident.